src/Microsoft.CodeAnalysis.ResxSourceGenerator/Microsoft.CodeAnalysis.ResxSourceGenerator.UnitTests/Resources/SingleString_RootNamespaceCSharpAsync/.Resources.Designer.cs

```diff
@@ -0,0 +1,21 @@
+﻿// <auto-generated/>
+
+#nullable enable
+using System.Reflection;
+
+
+namespace 
```

Comment:
This isn't legal C#, is it?

Reply:
It's not. The test is skipped as not currently supported.